tests/ui/test_discopane.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from pages.desktop.discopane import DiscoveryPane, DiscoveryPanePage, AboutAddons
+
+@pytest.fixture
+def firefox_profile(base_url, firefox_profile):
+    if 'allizom' in base_url:
+        # set the appropriate signatures for dev and staging
+        firefox_profile.set_preference('xpinstall.signatures.dev-root', True)
```

Comment:
What FF version are you using for tests. Afaik these prefs are not going to work for stable or beta from 48b5 onwards. See https://bugzilla.mozilla.org/show_bug.cgi?id=1186522